src/test/java/Problem2Test.java

```diff
@@ -9,11 +9,15 @@ public void testRemoveDuplicates() {
                 // DO NOT REMOVE OR CHANGE THE GIVEN TEST CASES!
                 {0, 0, 1, 1, 1, 2, 2, 4, 4, 6}
                 // add more test cases
+                , {10, 10, 20, 20, 20, 30, 30, 40, 60}
+                , {1, 1, 1, 1, 2, 2, 3, 4}
```

Comment:
Recommendation for future assignments: add edge cases (ex. when an object is NULL or empty) to verify your functions work for those too.